packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin.spec

```diff
@@ -61,14 +63,22 @@ install -p -m 0755 fips/nvidia-device-plugin %{buildroot}%{_cross_fips_bindir}
 
 install -d %{buildroot}%{_cross_unitdir}
 install -p -m 0644 %{S:1} %{buildroot}%{_cross_unitdir}
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -D -p -m 0644 %{S:2} %{buildroot}%{_cross_templatedir}/nvidia-k8s-device-plugin-conf
+install -m 0644 %{S:3} %{buildroot}%{_cross_tmpfilesdir}/nvidia-k8s-device-plugin-tmpfiles.conf
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia-k8s-device-plugin
 
 %files
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/nvidia-k8s-device-plugin
 %license LICENSE
 %{_cross_attribution_file}
 %{_cross_unitdir}/nvidia-k8s-device-plugin.service
 
 %files bin
 %{_cross_bindir}/nvidia-device-plugin
+%{_cross_templatedir}/nvidia-k8s-device-plugin-conf
+%{_cross_tmpfilesdir}/nvidia-k8s-device-plugin-tmpfiles.conf
```

Comment:
These should be part of the main package, not the `bin` package. Otherwise, when `fips` is enabled they won't be installed.